sktime/tests/test_softdeps.py

```diff
@@ -325,3 +326,45 @@ def test_est_fit_without_modulenotfound(estimator):
             f'to the "python_dependencies" tag, and python version bounds should be'
             f' added to the "python_version" tag. Exception text: {error_msg}'
         ) from e
+
+
+@patch("sktime.utils.dependencies._dependencies.sys")
+@pytest.mark.parametrize(
+    "mock_release_version, prereleases, expect_exception",
+    [
+        (True, True, False),
+        (True, False, True),
+        (False, False, False),
+    ],
```

Comment:
I'm tempted to suggest add the 4th possibility of (False, True, False) case, but I guess @fkiraly may want to reduce number of tests with total time concern.

Reply:
I added because it's extremely fast, just one function call that act upon a dictionary.

Reply:
no runtime concern on this one from my side...